tests/e2e/Services/Teams/TeamsBaseClient.php

```diff
@@ -292,9 +292,9 @@ public function testCreateTeamMembership($data): array
         $this->assertEquals('Invitation to ' . $teamName . ' Team at ' . $this->getProject()['name'], $lastEmail['subject']);
 
         /**
-         * Test for FAILURE
+         * Test for resend
+         * SUCCESS
          */
-
```

Comment:
No need to make a test for resend success block here. you can just add a 1 line comment saying this tests resending the invite.